python/tvm/topi/vision/nms.py

```diff
@@ -213,7 +213,7 @@ def get_valid_counts(data, score_threshold=0, id_index=0, score_index=1):
     out_indices: tvm.te.Tensor or numpy NDArray
         Related index in input data.
     """
-    if isinstance(score_threshold, float):
+    if isinstance(score_threshold, (float, int)):
```

Comment:
Just curious what's the scenario for score_threshold to be an int.

Reply:
I remember it is from https://github.com/apache/tvm/blob/3144cec7fac88707f0a5e338a5157435be070d6b/tests/python/topi/python/test_topi_vision.py#L131

It is only for passing this test. We can also change `0` to `0.0` instead.